spyder/plugins/ipythonconsole/utils/manager.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2009- Spyder Project Contributors
+#
+# Distributed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+# -----------------------------------------------------------------------------
+
+"""Kernel Manager subclass."""
+
+# Standard library imports
+import os
+import signal
+import sys
+
+# Third party imports
+from qtconsole.manager import QtKernelManager
+import psutil
+
+
+class SpyderKernelManager(QtKernelManager):

```

Comment:
Please add a docstring for this class and add below comments above the lines you added to make this work with our activation scripts.

Reply:
Fixed